packages/App/src/store/selectors/userIsLoading.tsx

```diff
@@ -0,0 +1,3 @@
+export const IsLoadingUser = (state: any) => {
```

Comment:
It will be more convenient to have all selectors in a file called by the name of reducer they serve to. It can be `registration` in your case.